pkgs/applications/graphics/dungeondraft/default.nix

```
@@ -0,0 +1,77 @@
+{ stdenv
+, lib
+, requireFile
+, dpkg
+, xorg
+, libGL
+, udev
+, pulseaudio
+, libkrb5
+, zlib
+}:
```

Consider formatting like this (still needs to be sorted according to order of use except lib).
```suggestion
{
  stdenv,
  lib,
  requireFile,
  dpkg,
  xorg,
  libGL,
  udev,
  pulseaudio,
  libkrb5,
  zlib,
}:
```

---

By the same syntax guide and the other packages I've looked at, the formatting I had should be right.
Is this a newer standard that hasn't been published online and the other packages haven't been updated to yet?

---

> By the same syntax guide and the other packages I've looked at, the formatting I had should be right. Is this a newer standard that hasn't been published online and the other packages haven't been updated to yet?

Ultimately this isn't blocking, this is just a suggestion. This is how [alejandra](https://github.com/kamadorueda/alejandra) would format it.